player.h

```diff
@@ -150,14 +154,28 @@ int playersCollisionWithOtherPlayers(int playerCount){
 	return -1;
 }
 
+int playerCollisionWithOtherPlayers(int playerCount,int playerToTest){
+	for (int i = 0; i < playerCount; i++){
+		if (players[i].isAlive){
+			if (i != playerToTest){
+				if (players[i].x == players[playerToTest].x && players[i].y == players[playerToTest].y){
+					return i;
+				}
+			}
+		}
+	}
+	return -1;
```

Comment:
Please, do not use magic numbers :smile:

Reply:
Done! 😄